plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/TestProject.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.localserver.server;
+package com.google.cloud.tools.eclipse.test.util.project;
```

Comment:
I've changed the name to `TestProjectCreator`. I'm against `FacetedProjectCreator` only because we can create non-faceted projects with this class as well.